README.md

```diff
@@ -50,22 +62,22 @@ pipeline:
     observationTimeStamp: '2018-10-11T13:01:02-08:00'
 ```
 
-Step 1 involves copying the source files into a folder:
+First we copy all the source files into a folder:
 ```bash
-mkdir ~/inputs
-cp orthomosaic.tif ~/inputs
-cp plot_shapes.* ~/inputs
-cp experiment.yaml ~/inputs
+mkdir /inputs
+cp orthomosaic.tif /inputs
+cp plot_shapes.* /inputs
+cp experiment.yaml /inputs
 ```
 
-In step 2 we create an folder to hold the output of our processing:
+Next we create an folder to hold the output of our processing:
 ```bash
-mkdir ~/output
+mkdir /output
 ``` 
 
 Finally we run the container mounting our source and destination folders, as well as indicating the name of the orthomosaic file and the name of the shapefile.
 ```bash
-docker run --rm -v ~/inputs:/scif/data/odm/images -v ~/outputs:/output agdrone/canopycover-workflow:latest run short_workflow orthomosaic plot_shapes.shp
+docker run --rm -v /inputs:/scif/data/odm_workflow/images -v /outputs:/output agdrone/canopycover-workflow:latest run short_workflow orthomosaic plot_shapes.shp
```

Comment:
Problem: Fell in to the trap I mentioned above. The instruction says to create a directory called `/output`, but we run the `docker run` with `/outputs`.

- [ ] Fix this (blocking)

Reply:
Fixed